pkg/ims/repository/postgresql/postgres.go

```diff
@@ -0,0 +1,141 @@
+package postgresql
```

Comment:
as log as this file contains only users handlers, it is better to rename it to `users_repository.go`